addons/event/models/event.py

```diff
@@ -348,6 +348,15 @@ def write(self, vals):
         res = super(EventEvent, self).write(vals)
         if vals.get('organizer_id'):
             self.message_subscribe([vals['organizer_id']])
+        if 'stage_id' in vals:
+            for event in self:
+                event_mail_ids = event.event_mail_ids.filtered(
+                    lambda event_mail:
+                    event_mail.interval_type == 'stage_update'
+                    and not event_mail.on_update_stage_date
+                    and event_mail.on_update_stage_id.id == event.stage_id.id)
+                if event_mail_ids:
+                    event_mail_ids.on_update_stage_date = fields.Datetime.now()
```

Comment:
@dbeguin 
I think it can do weird things if we use a computed field because we are in the write method... (will the computed field always be "invalidate" after the super ?)